### ISSUE TITLE
Feature request: Add the option to run Localsend in the Backgroun on Android devices

### ISSUE THREAD

**Issue by DraconicNEO** [localsend/localsend#2153]
### Is your feature request related to a problem? Please describe.

On the Android version of the app I cannot seem to be able to run the app in the background which is a pitty since it is rather useful to keep it in the background ready to use, just like I do in the Flatpak version on my Desktop.

### Describe the solution you'd like

Add an option to keep Localsend on Android running in the background, preferably with a toggle tile in the QuickSettings menu to enable Background mode or disable it, and have it prompt with an overlay if you try to send a file to that device.

### Describe alternatives you've considered

N/A

### Additional context

I'm using the apk from the Github release, So if this feature is present in the Play store releases I wouldn't know.

**Comment by notacodermadeaccountforreportissue** [comment]
Commenting for same.

**Comment by AzubuikeKevin** [comment]
I second this issue.
Feature for running in the background should be considered

**Comment by WeareTheDraiken** [comment]
Yeah kinda weird the app can't recieve in Background.

**Comment by SamConners47** [comment]
Was about to open up an issue for it myself. I wish for the same.

**Comment by Zero476195** [comment]
I agree, this is definitely an issue

**Comment by scrapp08** [comment]
Bumping this, crucial feature.

**Comment by KhalilOuali** [comment]
Agreed as well. Would be much more convenient. Along with notifications and/or overlays.

**Comment by kshitijpatil** [comment]
Bumping this again. Is this being considered for future release?

**Comment by mvrvelous** [comment]
> Bumping this again. Is this being considered for future release?

Coincidence, you just posted this. I've always wanted this feature but it seems it's not a priority or even considered at all.
I don't want to be an a55 but Syncthing is a nice alternative. But what I also like about Localsend is the paste feature. With Syncthing I would have to paste the text into a text file first.

**Comment by SamConners47** [comment]
> > Bumping this again. Is this being considered for future release?
> 
> Coincidence, you just posted this. I've always wanted this feature but it seems it's not a priority or even considered at all.
> I don't want to be an a55 but Syncthing is a nice alternative. But what I also like about Localsend is the paste feature. With Syncthing I would have to paste the text into a text file first.

I agree. I also wish to share an alternative - Blip. It works quite seamlessly. I have been using it for a couple of months now and seem to have found no issue so far. Hope this helps!
